text.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="logo">Jo</div>
+            <div></div>
+            <div class="menu">
+                <a href="#" class="nav">
+                    <div ></div>
+                    <div ></div>
+                    <div ></div>
+                </a>
+            </div>
+        </header>
+        <section class="headline">
+            <div></div>
+            <div class="greet">I'm Jonathan Glad to see you!</div> 
```

Comment:
- [ ] if you want that google (and other search engines) know about a specific topic, in this case, you, use a `<h1>` tag for your introductory text instead of `<div class="greet">I'm Jonathan Glad to see you!</div>`. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.